auth/login.html

```diff
@@ -1,15 +1,21 @@
 {{define "content"}}
-	<h2>Login</h2>
-	<form method="post">
-		<div class="form-group">
-			<label for="username">User</label>
-			<input class="form-control" type="text" id="username" name="username" />
+<div class="container">
+		<h1 class="welcome">Reach your community</h1>
+		<h1 class="welcome">with <span class="luckdrum">LuckDrum</span></h1>
+		<p class="desc">Earth's GLOCAL broadcasting drum and message wall</p>
+		<div class="login-container col-md-4">
+			<h3>Log in</h3>	
```

Comment:
I see your point, but semantics says headings shouldn't be skipped: http://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/G141

> To facilitate navigation and understanding of overall document structure, authors should use headings that are properly nested (e.g., h1 followed by h2, h2 followed by h2 or h3, h3 followed by h3 or h4, etc.).